app/src/main/java/org/cru/godtools/ui/tools/ToolsContainerFragment.kt

```diff
@@ -0,0 +1,19 @@
+package org.cru.godtools.ui.tools
+
+import android.os.Bundle
+import androidx.fragment.app.viewModels
+import dagger.hilt.android.AndroidEntryPoint
+import org.cru.godtools.R
+import org.cru.godtools.databinding.ToolsContainerFragmentBinding
+import org.cru.godtools.fragment.BasePlatformFragment
+
+@AndroidEntryPoint
+class ToolsContainerFragment : BasePlatformFragment<ToolsContainerFragmentBinding>(R.layout.tools_container_fragment) {
```

Comment:
let's name this either: `ToolsFragment` or `DashboardToolsFragment`.

I'm leaning more towards the first one, but I know it will have some temporary import conflicts as you try to use it within `MainActivity`. I think we will probably want to rename the pre-existing `ToolsFragment` to be `ToolsListFragment` (don't do that in this PR, we will address that later).